src/tests/system/tests/test_authentication.py

```diff
@@ -28,6 +33,8 @@ def test_authentication__login(client: Client, provider: GenericProvider, method
         2. User is not authenticated
     :customerscenario: False
     """
+    client.sssd.set_service_user(sssd_service_user)
```

Comment:
I think this line should go together with the call to `start()`, as you did in lines 53~56.

Reply:
Moved.